CONTRIBUTING.rst

```diff
@@ -4,11 +4,23 @@
 Contributing
 ============
 
+If you're not sure this is the correct place to file an issue
+then please file an issue on the `mozilla/addons`_ project instead.
+
 Before contributing code, please note:
 
-* You agree to license your contributions under the [license](https://github.com/mozilla/addons-server/blob/master/LICENSE).
-* Ask on the [dev-addons mailing list](https://mail.mozilla.org/listinfo/dev-addons) or in GitHub before making changes.
-* Follow [PEP8](http://www.python.org/dev/peps/pep-0008/), [jshint](http://www.jshint.com/) and our other [style guide conventions](http://mozweb.readthedocs.org/en/latest/index.html).
-* Please write tests and read the docs on [addons-server](http://addons-server.readthedocs.org/en/latest/).
+-  You agree to license your contributions under the `license`_.
+-  Ask on the `dev-addons mailing list`_ or in GitHub before making
```

Comment:
Not sure what exactly `in GitHub` refers to? Let's clarify? (I know that didn't change, but while we're at it...)